docs/visualize/most-frequent.asciidoc

```diff
@@ -1,57 +1,54 @@
 [[most-frequent]]
-== Most frequently used visualizations
+== Classic visualizations
 
-The most frequently used visualizations allow you to plot aggregated data from a <<save-open-search, saved search>> or <<index-patterns, index pattern>>.
+Visualize includes several classic visualizations that allow you to plot, visualize, and analyze your aggregated Elasticsearch data.
```

Comment:
Are these also in Lens?

Visualize should be in bold here and elsewhere in the doc.